integrations/docker/images/chip-build-efr32/Dockerfile

```diff
@@ -29,9 +30,10 @@ RUN wget https://www.silabs.com/documents/login/software/slc_cli_linux.zip && \
 
 ENV PATH="${PATH}:/slc_cli/"
 
-#  TODO Fix me
-# RUN slc configuration  --sdk="$GSDK_ROOT" && \
-#     slc signature trust --sdk "$GSDK_ROOT"
-
+# Install Python Packages
 
+# SLC required Python Packages
```

Comment:
these are not required by SLC, my guess is these are codegen.py requirements by the looks of it (happen to be familiar with that tool, these are exactly its dependencies).